client/project_policy.go

```diff
@@ -13,23 +13,23 @@ type Policy struct {
 	UpdatedBy                   string  `json:"updatedBy"`
 	RunPullRequestPlanDefault   bool    `json:"runPullRequestPlanDefault"`
 	ContinuousDeploymentDefault bool    `json:"continuousDeploymentDefault"`
-	MaxTtl                      *string `json:"maxTtl,omitempty" tfschema:",omitempty"`
-	DefaultTtl                  *string `json:"defaultTtl,omitempty" tfschema:",omitempty"`
+	MaxTtl                      *string `json:"maxTtl" tfschema:",omitempty"`
```

Comment:
unfortunately, the API needs to pass "null" explicitly therefore, I had to remove the omitempty.
This is incorrect behavior in the backend (Ok for PATCH requests but wrong for PUT requests).